lib/tasks/temp.rake

```diff
@@ -1,6 +1,16 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Convert serialized params_yaml from PostgreSQL::OID::Integer to ActiveModel::Type::Integer'
+  task :update_params_yaml => :environment do
+    InfoRequestEvent.where("params_yaml LIKE '%OID::Integer%'").find_each do |event|
+      new_params =
+        event.params_yaml.gsub!('!ruby/object:ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Integer',
```

Comment:
Probably want `gsub` so we don't set the params to nil if no substitutions are made.

Reply:
good point, thanks!